### PR TITLE
make profile modal content scrollable

### PR DESCRIPTION
**Pull request by nick97eagles**
Size of modal is now static, but the body of the modal is now scrollable. 
Related to issue #88

### CODE REVIEW

File: src/app/shared/profile-modal/profile-modal.component.css [hunk 1]
```diff
@@ -10,3 +10,27 @@
...
+::-webkit-scrollbar {
+  background-color: black; /* removes scrollbar space*/
+}
+
+::-webkit-scrollbar-thumb {
+  background: gray;
+}
+
+/* removes horizontal line */
+.modal-header {
```
Comment (ermsdev) [inline src/app/shared/profile-modal/profile-modal.component.css]: combine these into one block

File: src/app/shared/profile-modal/profile-modal.component.css [hunk 2]
```diff
@@ -10,3 +10,27 @@
 .modal-backdrop {
   background-color: rgb(20, 20, 20);
 }
+
+.modal-body {
+  max-height: calc(100vh - 130px);
```
Comment (ermsdev) [inline src/app/shared/profile-modal/profile-modal.component.css]: how does this look without the 130px? can we avoid using specific pixel values?

Comment (nick97eagles) [reply src/app/shared/profile-modal/profile-modal.component.css]: We looked over this and got it looking great for both desktop and mobile

File: src/app/shared/profile-modal/profile-modal.component.css (hunk 1 above)
Comment (nick97eagles) [reply src/app/shared/profile-modal/profile-modal.component.css]: done 👍🏻

**Pull request merged**